_pages/learning/crystal_programming.html

```diff
@@ -1,81 +1,124 @@
 ---
-title: "Book: Crystal Programming"
-description: |
-  <strong>Crystal Programming</strong> is a project-based book by George Dietrich and Guilherme Bernal that covers all the
-  fundamental concepts needed to start programming in Crystal, such as OOP and concurrent programming.
-layout: learning
+title: "Building efficient, safe, and readable web and CLI applications with Crystal"
+layout: wide
+---
 
-headline: Building efficient, safe, and readable web and CLI applications with Crystal
-cover_image: /assets/learning/crystal_programming.png
-vendors:
-- url: https://www.amazon.com/Crystal-Programming-project-based-introduction-applications/dp/1801818673/ref=sr_1_2
-  name: Amazon
-- url: https://www.packtpub.com/product/crystal-programming/9781801818674
-  name: Packt
+<div class="book-presenter">
+  <div class="book-presenter__main">
+    <div class="book-presenter__cover">
+      <img src="/assets/learning/crystal_programming.png" />
+    </div>
 
-book_info:
-  name: Crystal Programming
-  isbn: 978-1801818674
-  publisher: Packt Publishing
-  numberOfPages: 356
-  datePublished: 2022-05-27
+    <div class="book-presenter__description">
+      <summary>
+        <strong>Crystal Programming</strong> is a project-based book by George Dietrich and Guilherme Bernal that covers all the
+        fundamental concepts needed to start programming in Crystal, such as OOP and concurrent programming.
+      </summary>
```

Comment:
This is not correct usage of the `<summary>` element. It's only valid inside a `<details>` element. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary

If your intention is to make semantic annotations: I don't think we have much of that currently, but I'd appreciate it very.
This can be expressed using microdata for https://schema.org/Book (this needs to go along with an itemscope definition and other metadata of course).
```suggestion
      <p itemprop="abstract">
        <strong>Crystal Programming</strong> is a project-based book by George Dietrich and Guilherme Bernal that covers all the
        fundamental concepts needed to start programming in Crystal, such as OOP and concurrent programming.
      </p>
```